README.md

```diff
@@ -132,6 +132,22 @@ This generator creates an example home controller and view which fetches and dis
 Options include:
 * __[Not recommended for embedded apps]__ `with-cookie-authentication` - This flag generates an authenticated home_controller, where the authentication strategy relies on cookies. By default, this generator creates an unauthenticated home_controller and protected sample products_controller.
 
+### Scopes Storage Generator
```

Comment:
Not at all related to this PR, but this README is starting to feel too long to me - do we want to look into breaking it up into smaller pages?

Reply:
We have an issue that will look to break this README up very soon. Always looking for suggestions on how to best split this doc up!

Reply:
Yes! We could probably break down the sections into different files?